crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -55,6 +57,11 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    /// Create a new `Commands` from a queue and an [`Entities`] reference.
+    pub fn new_from_entities(queue: &'s mut CommandQueue, entities: &'w Entities) -> Self {
```

Comment:
Can we get a quick doc example of how you'd actually access this in e.g. a test?

Reply:
Unfortunately, it seems that so far this method cannot be effectively used, as `SystemParam` derives cannot currently implement custom apply strategies, and I don't think it'd be a great idea to add an entire manual `SystemParam` impl here to use as an example. I'll make a separate PR for custom apply strategies in `SystemParam` derives later on, which could then be used as an example here.